src/communityFeed/api/communityFeed.js

```diff
@@ -3,14 +3,17 @@ import axios from '@/base/api/axios'
 const backend = process.env.MODE === 'cordova' ? process.env.KARROT.BACKEND : ''
 
 export default {
+  async getTopic (id) {
+    return (await axios.get(`/community_proxy/t/${id}.json`)).data
+  },
   async getMeta () {
     return convert((await axios.get('/api/community-feed/')).data)
   },
   async markSeen () {
     return (await axios.post('/api/community-feed/mark_seen/')).data
   },
   async latestTopics () {
-    const data = (await axios.get('/community_proxy/c/karrot.json')).data
+    const data = (await axios.get('/community_proxy/c/karrot/7.json')).data
```

Comment:
Is this the Karrot category?

Reply:
Yes, `'/community_proxy/c/karrot.json'` redirects to `/community_proxy/c/karrot/7.json` ... but with the full URL, and CORS blocks it in local dev.